### ISSUE TITLE
How to call a callback function in a page in a supertab?

### ISSUE THREAD

**Issue by akramaskar** [zyra/ionic-super-tabs#411]
Hi everyone!

I have an issue regarding on how to call a callback function in a page in a super-tab
This is an example of such a page:

`export class PickDegreePage {
  @Output() onSearch: EventEmitter<any> = new EventEmitter();

  data: any = {};
  rootNavCtrl: NavController;

  constructor(
    public navCtrl: NavController,
    public navParams: NavParams,
  ) {
    this.data = this.navParams.data;
  }

  
  search(event) {
    console.log("Search pick degree", event);
    this.onSearch.emit(event);
  }`

And this is how it looks like in the super-tab tag on the parent page:

`<super-tab
              *ngFor="let page of pages"
              [root]="page.pageName"
              [title]="page.title"
              [rootParams]="data"
              [id]="page.id"
              (onSearch)="search($event)"
            >
</super-tab>`

Basically, I want to call the search function which I have already defined on the parent page when the search is also happening on the child page.

Do I miss something?

**Comment by ihadeed** [comment]
Your output is on the inner component and not on the actual super tab component.

What version of Super Tabs are you using?

v5: you need to pass a reference to the function via the `rootParams` data. Then you can call `onSearch` on the root page.

v6+: you can have a sub component under the `<super-tab>` bind to `onSearch`

The usage you have above is invalid for all  versions of super tabs.
